src/main/java/org/example/FlashcardManager.java

```diff
@@ -18,6 +18,8 @@ public class FlashcardManager {
     public FlashcardManager(){
                 this.flashcards = new HashMap<>();
             }
+            
+            
 
     public void addCard(String term, String definition,int mistakes){
```

Comment:
Trata de agregar validaciones a los inputs para evitar errores en el codigo 
```suggestion
    public void addCard(String term, String definition, int mistakes){
        if (term == null || definition == null) {
            throw new IllegalArgumentException("Term and definition cannot be null");
        }
        if (mistakes < 0) {
            throw new IllegalArgumentException("Mistakes cannot be negative");
        }
        Flashcard card = new Flashcard(term, definition, mistakes);
        flashcards.put(term, card);
    }
```